steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml

```diff
@@ -0,0 +1,73 @@
+title: Build test bundle for AWS Device Farm
+summary: Build your XCTest or XCUITest bundle to upload to AWS Device Farm
```

Comment:
I'd also just say "for example AWS device far" as it should work with other services too